### DISCUSSION TITLE
Active players show in phone notifications panel

### DISCUSSION THREAD

**Discussion by patienttruth** [music-assistant/support#2345] [Feature requests and ideas]
Similar to how local Chromecasts show up in your notification panel, MA would show up. This would make it possible to quickly swipe and pause or skip from the panel or even lock screen of a device.

I'm guessing it would take emulating a Chromecast's network discovery that gets picked up by a phone.

![Screenshot_20240520-114704](https://github.com/music-assistant/support/assets/105557996/cc2b830a-f6bf-411a-b6c1-51c13426a6da)

Thanks everyone who makes MA awesome!

**Comment by OzGav** [comment]
Probably this will happen when/if phone apps get developed.

**Reply by patienttruth** [reply]
That makes sense. 

Though, I don't think this necessarily requires an app to be installed? I believe it's got to do with Google Play services on the phone discovering a device casting. It may not be practical, as it probably has to do with somehow impersonating or replicating the cast protocol discovery messages.

**Reply by patienttruth** [reply]
My screenshot is not actually the best example I believe that section is driven by an app. I wasn't thinking through properly when I took that screenshot.

The discovery part I'm referring to actually shows up in the regular notification area. I'll have to send a different screenshot. When I open up the settings on this particular notification it goes into the Google Play services notification configuration.

I'm not at home to make an example of what the notification looks like, but this is the configuration panel indicating that it's driven by Google Play services.

![Screenshot_20240522-120238.png](https://github.com/music-assistant/support/assets/105557996/abf44da3-a414-4f95-bd09-62356f65fc81)

**Reply by patienttruth** [reply]
This popped up when I rolled into the driveway.

It doesn't contain all the details about what's playing since this phone didn't initiate the cast, it's just showing an active cast on the network.

I thought it gave volume control automatically, but it looks like play/pause/skip/stop, and mute are all you get.

![Screenshot_20240522-120648.png](https://github.com/music-assistant/support/assets/105557996/843b0a16-fdb5-45c3-bc1b-0a837b22b108)
